src/clj/rems/workflow/dynamic.clj

```diff
@@ -827,6 +902,61 @@
                               [{:type ::invite-member :actor "assistant" :member {:name "Member Applicant 1" :email "member1@applicants.com"}}]
                               injections)))))))
 
+(deftest test-remove-member
+  (let [application {:state ::submitted
+                     :members [{:userid "applicant"} {:userid "somebody"}]
+                     :applicantuserid "applicant"
+                     :workflow {:type :workflow/dynamic
+                                :handlers ["assistant"]}}
```

Comment:
Please refactor this so that the application is produced from events with `apply-events`, similar to all the other tests.

Reply:
Ok I based my tests on earlier code that didn't use the events.